human_aware_rl/human/process_human_trials.py

```diff
@@ -1,6 +1,6 @@
 import pickle
```

Comment:
@mesutyang97 I converted you file into an argparse script and added a corresponding unit test. However, the thing breaks on the dummy dataset I included for testing purposes (its a random subsample of the 2019 neurips pickle data). 

Unfortunately, on github, all the data-oriented tests are throwing a key error, which tells me there's something wrong with the CI checkout process (maybe because the pickle files are almost near the max github file size). Therefore, you won't be able to see the error I'm referring to in the integration check. I've thus included a screenshot of the error from my local tests. I doubt the screenshot will be super enlightening, so hopefully we can figure out both the github CI issues and then more efficiently tackle this bug


<img width="579" alt="Screen Shot 2021-05-25 at 3 30 34 PM" src="https://user-images.githubusercontent.com/35500370/119576775-3f3dd080-bd6e-11eb-88bf-3af4e68efae8.png">

Reply:
I think this is because when I make the conversion, if player doesn't hold an object, the PlayerState simply won't have the "held_object" key. But if we want to stay consistent and always have that key (and just have the value be None when the player isn't holding an object), we can simply add a checking condition in the process_held_object function. 

Let me know what's the desired behavior, and I will update this file accordingly

Reply:
Yeah not sure if you saw my later follow-ups, but the issue was a schema mismatch. The old schema would simply not have a `held_objects` key if the player wasn't holding anything, the new schema has `held_object : None` in that case. 

I ported __all__ of the data in the repo to the new schema, and, just to be safe for legacy reasons, updated your script so that it can handle both legacy and current schema, depending on whether the `forward-port` flag is passed in or not